security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -1023,7 +1023,9 @@ func (ms *mockSecretStore) DeleteSecret(conID, resourceName string) {
 }
 
 func (ms *mockSecretStore) ShouldWaitForIngressGatewaySecret(connectionID, resourceName, token string) bool {
-	return false
+	// Return true here for file mounted cert resource name to validate that file mounted certs would
```

Comment:
I am unsure that the comment added here is accurate: "// Return true here for file mounted cert resource name to validate that file mounted certs would not depend on this result and proceed with generating secret.".
1. What result would not be depended?
2. Can you revise the aforementioned comment to make its meaning easier to understand (e.g., break this long sentence into multiple shorter sentences; and explain 1) at which scenario which resources will be waited, 2) at which scenarios which resources will not be waited)?

Reply:
Is this comment clear now? or do you want me to add any thing more?

Reply:
According to the code, when  strings.HasPrefix(resourceName, "file-root:") returns true, ShouldWaitForIngressGatewaySecret() returns true, which means waiting for ingress gateway secret.
Can you explain why waiting for ingress gateway secret or not is determined by the "file-root" resource?

Reply:
With the new change this check not needed because it moves to ShouldWaitForIngressGatewaySecret. I will remove it